helm/ida-auth/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: ida-auth
```

Comment:
id-authentication/deploy/ida
/install.sh

CHART_VERSION=1.3.0-beta.2-develop to CHART_VERSION=1.3.x